src/main/java/org/sagebionetworks/bridge/workerPlatform/bridge/AccountSummaryIterator.java

```diff
@@ -38,12 +40,14 @@ public AccountSummaryIterator(ClientManager clientManager, String studyId) {
     private void loadNextPage() {
         // Call server for the next page.
         try {
-            // The offset into the next page is equal to the number of accounts that we have seen.
             // HACK: We use "1" for the phone filter. This is because we want to filter out any accounts that don't
             // have phone numbers. Right now, all phone accounts are in the US, so we can simply use the country code
             // ("1") in the filter.
```

Comment:
FWIW there are some Indian phone numbers in the system.

Reply:
Updated the comment. We only support sending to US phone numbers. (Or at least the code as written will only send to phone numbers in the North America Numbering Plan. If we want to support mPower 2.0 to send reminder notifications to non-US numbers, that's a separate discussion and a separate code change.